<launch>
  <arg name="trajectory_controller_name" />
  <arg name="name_prefix" default="right_hand" />

  <!-- Load controller settings -->
  <rosparam file="$(find schunk_svh_driver)/cfg/$(arg name_prefix)_controllers.yaml" command="load"/>

  <!-- spawn controller manager with a timeout to give enough time to finger reset -->
  <node name="hand_ros_control_controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
    output="screen" args="--timeout 90 joint_state_controller $(arg trajectory_controller_name)" />

</launch>
